README.md

```diff
@@ -18,40 +18,113 @@ Feel free to explore, **contribute**, and make the most of this repository as yo
 ## Programmes
 
 Click to expand the list of courses for each programme.
-
 <details>
-<summary><b>&#x1F447; Software Engineering and Management BSc.</b></summary>
+<summary><b>&#x1F447; N1SOF - Software Engineering and Management Bachelor's Programme</b></summary>
```

Comment:
LGTM for now! Undeniably, the `README.md` file may need some polishing, but that will be a **separate issue**. We'll need to indicate _pre-2024_/ _post-2024_ revisions of the BSc. SEM programme.
Good job.